File: Assignment 2/if else/addmul.cpp
#include<iostream>
using namespace std;
class Add_Multiply

{
    int num1,num2,result1,result2;
    public:void great()
    {
        cout<<"Enter any two numbers that you want to add:\n";
        cin>>num1>>num2;
        result1=num1+num2;
        cout<<"The result of addition is: "<<result1;
        cout<<endl;
        result2=num1*num2;
        cout<<"The result of multiplication is:"<<result2;
        cout<<endl;
        if(result1>result2)
        {
            cout<<"The result of addition is greatest\n"<<result1<<endl;

        }
        else
        {
            cout<<"The result of multiplication is greatest\n"<<result2;
        }


    }
};
int main()
{
    Add_Multiply obj;
    obj.great();
}

File: Java assignment(1-2)/java Assignment 2/Add_Mul.java
import java.util.Scanner;
class Add_Mul
{
float num1,num2,add,mul;
void accept()
{
Scanner sc=new Scanner(System.in);
System.out.println("Enter two numbers: ");
num1=sc.nextFloat();
num2=sc.nextFloat();
}
void display()
{
add=num1+num2;
mul=num1*num2;
if(add>mul)
{
System.out.println("Addition of the two values is the greatest");
}
else
{
System.out.println("Multiplication of the two values is the greatest");
}
}
public static void main(String[] args)
{
Add_Mul obj=new Add_Mul();
obj.accept();
obj.display();
}
}